offload/tools/offload-tblgen/offload-tblgen.cpp

```diff
@@ -44,6 +45,8 @@ cl::opt<ActionType> Action(
         clEnumValN(DumpJSON, "dump-json",
                    "Dump all records as machine-readable JSON"),
         clEnumValN(GenAPI, "gen-api", "Generate Offload API header contents"),
+        clEnumValN(GenSpec, "gen-spec",
```

Comment:
Do we want to call it a "spec" here? Is it a spec or a document? I think they are different.